src/constants/logURLTemplates.ts

```diff
@@ -52,8 +52,8 @@ const getResmokeLogURL = (
   return `${logkeeperURL}/build/${buildID}/all?${stringifyQuery(params)}`;
 };
 
-enum originToType {
-  agent = "A",
+enum OriginToType {
+  agent = "E",
   system = "S",
   task = "T",
```

Comment:
```suggestion
  Agent = "E",
  System = "S",
  Task = "T",
```

Reply:
i can't actually do this since the incoming URLs are lower cased letters.

Reply:
https://jira.mongodb.org/browse/EVG-18667